notebooks/02-ingestion/12-spatial-dimensions.ipynb

```diff
@@ -0,0 +1,709 @@
+    "      \"type\": \"index_parallel\",\n",
+    "      \"inputSource\": {\n",
+    "        \"type\": \"http\",\n",
+    "        \"uris\": [\n",
+    "          \"http://datagen:9999/file/users.json\"\n",
+    "        ]\n",
+    "      },  \n",
+    "      \"inputFormat\": {\n",
+    "        \"type\": \"json\"\n",
+    "      },\n",
+    "      \"appendToExisting\": False\n",
+    "    },\n",
+    "    \"dataSchema\": {\n",
+    "      \"granularitySpec\": {\n",
+    "        \"segmentGranularity\": \"day\",\n",
+    "        \"queryGranularity\": \"none\",\n",
+    "        \"rollup\": False\n",
+    "      },\n",
+    "      \"dataSource\": \"example-spatial-index\",\n",
+    "      \"timestampSpec\": {\n",
+    "        \"column\": \"time\",\n",
+    "        \"format\": \"iso\"\n",
+    "      },\n",
+    "      \"dimensionsSpec\": {\n",
+    "        \"dimensions\": [\n",
+    "          \"user_id\",\n",
+    "          \"first_name\",\n",
+    "          \"last_name\",\n",
+    "          \"dob\",\n",
+    "          {\n",
+    "            \"name\": \"address_lat\",\n",
+    "            \"type\": \"double\"\n",
+    "          },\n",
+    "          {\n",
+    "            \"name\": \"address_long\",\n",
+    "            \"type\": \"double\"\n",
+    "          },\n",
+    "          \"marital_status\",\n",
+    "          {\n",
+    "            \"name\": \"income\",\n",
+    "            \"type\": \"double\"\n",
+    "          },\n",
+    "          \"signup_ts\"\n",
+    "        ],\n",
+    "        \"spatialDimensions\": [\n",
+    "          {\n",
+    "             \"dimName\": \"address_spatial\",\n",
+    "             \"dims\": [\n",
+    "               \"address_lat\",\n",
+    "               \"address_long\"\n",
+    "             ]\n",
+    "          }\n",
+    "        ]\n",
+    "      }\n",
+    "    },\n",
+    "    \"tuningConfig\": {\n",
+    "      \"type\": \"index_parallel\",\n",
+    "      \"partitionsSpec\": {\n",
+    "        \"type\": \"dynamic\"\n",
+    "      }\n",
+    "    }\n",
+    "  }\n",
+    "}\n",
+    "headers = {\n",
+    "  'Content-Type': 'application/json'\n",
+    "}\n",
+    "\n",
+    "task = rest_client.post(\"/druid/indexer/v1/task\", json.dumps(spatial_index_spec), headers=headers)\n",
+    "task_id = json.loads(task.text)['task']\n",
+    "wait_for_task(task_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bd176f7-c443-41ec-af7a-1e50d69fae01",
+   "metadata": {},
+   "source": [
+    "## Filter spatial data \n",
+    "Querying spatial data also requires the using indexed filtering currently requires the use of the native query API. \n",
+    "Spatial filters are specified as follows:\n",
+    "\n",
+    "```\n",
+    "\"filter\": {\n",
+    "  \"type\": \"spatial\",\n",
+    "  \"dimension\": <name_of_spatial_dimension>,\n",
+    "  \"bound\": <bound_type>\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "There are 3 types of bounds to find spatial points:\n",
```

Comment:
```suggestion
    "The following are the types of bounds for spatial points:\n",
```

Reply:
I will rewrite as "The following are the types of bounds for spatial filters:"

Reply:
done